### PR TITLE
Add NeuVector Monitor chart version 2.7.3

### PR DESCRIPTION
**Pull request by selvamt94**
## Problem
NeuVector Monitor chart 102.0.8+up2.7.3 need to be added to Rancher feature charts version 2.7 (release-v2.7). We would like to include this chart in Rancher Manager release v2.7.11 otherwise this need to be released as OOB.
## Solution
new chart has been added

### Manual Testing
Tested fresh install of the NeuVector Monitor chart 102.0.8+up2.7.3 and upgraded from previous version on RKE and RKE2 cluster.
## QA Testing Considerations
Tested fresh install of the NeuVector Monitor chart 102.0.8+up2.7.3 and  upgraded from previous version on RKE and RKE2 cluster.

### CODE REVIEW

File: charts/neuvector-monitor/102.0.8+up2.7.3/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: NeuVector Monitor
+  catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.29.0-0'
```
Comment (nicholasSUSE) [inline charts/neuvector-monitor/102.0.8+up2.7.3/Chart.yaml:4]: Should this be?
```
catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.28.0-0'
```

**Review by nicholasSUSE — CHANGES_REQUESTED** [review]

File: charts/neuvector-monitor/102.0.8+up2.7.3/Chart.yaml (hunk 1 above)
Comment (selvamt94) [reply charts/neuvector-monitor/102.0.8+up2.7.3/Chart.yaml:4]: @nicholasSUSE NeuVector supports k8s 1.28 so updated the annotation. I tested the chart on downstream cluster with k8s 1.28.6

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

File: charts/neuvector-monitor/102.0.8+up2.7.3/Chart.yaml (hunk 1 above)
Comment (lucasmlp) [reply charts/neuvector-monitor/102.0.8+up2.7.3/Chart.yaml:4]: @nicholasSUSE This chart might be used outside Rancher Manager. We offer all the charts via [charts.rancher.io](charts.rancher.io) so users can use them via `helm CLI`. It's up to the feature chart teams to say in which k8s versions the charts might be installed

**Review by nicholasSUSE — APPROVED** [review]

**Comment by nicholasSUSE** [comment]
@selvamt70 just resolve the conflicts, I will leave it approved

**Pull request merged**